data_juicer/ops/mapper/video_captioning_from_summarizer_mapper.py

```diff
@@ -81,6 +81,7 @@ def __init__(self,
         :param args: extra args
         :param kwargs: extra args
         """
+        kwargs.setdefault('mem_required', '40GB')
```

Comment:
Perhaps we may add new Fields like `mem_required_quant_8` for these large-model-based OPs